src/bin/pgcopydb/ld_stream.h

```diff
@@ -121,6 +123,7 @@ typedef struct StreamContext
 typedef struct StreamApplyContext
 {
 	CDCPaths paths;
+	LogicalStreamMode mode;
```

Comment:
I am not quite sure yet in this review why that's needed?

Reply:
Yeah I need to remove this. Earlier I thought we only need spilled txn for catch-up mode only, but I was wrong.